schema_based_on_diagram.sql.sql

```diff
@@ -1 +1,49 @@
-/**All db schema*/
\ No newline at end of file
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(250) NOT NULL,
+    date_of_birth DATE
+);
+
+CREATE TABLE history_treatment_id (
+  medical_history_fk INT PRIMARY KEY,
+  treatment_fk INT PRIMARY KEY
+);
```

Comment:
- [ ] These two should be foreign keys that reference the PRIMARY `ID` column of their respective table. KIndly implement it. 👍